docs/administration/deployment/upgrading-db/index.md

```diff
@@ -0,0 +1,110 @@
+following command:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -c rest-api -- reana-db alembic upgrade
+```
+
+The output will show the accomplished upgrades, for example:
+
+```console
+$ kubectl default exec -i -t deployment/reana-server -c rest-api -- reana-db alembic upgrade
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade  -> c912d4f1e1cc, Quota tables.
+INFO  [alembic.runtime.migration] Running upgrade c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+INFO  [alembic.runtime.migration] Running upgrade ad93dae04483 -> 4801b98f6408, Job started and finished times.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+INFO  [alembic.runtime.migration] Running upgrade f84e17bd6b18 -> 6568d7cb6710, storing full workflow workspace.
+```
+
+## Detailed upgrade procedure
+
+First, you can check all the alembic `history` to list all the
+existing revisions:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -c rest-api -- reana-db alembic history
+f84e17bd6b18 -> 6568d7cb6710 (head), storing full workflow workspace.
+4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+ad93dae04483 -> 4801b98f6408, Job started and finished times.
+c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+<base> -> c912d4f1e1cc, Quota tables.
+```
+
+And also, get the exact hash of your `current` revision:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -c rest-api -- reana-db alembic current
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+4801b98f6408
+```
+
+In this particular example, the current revision hash is `4801b98f6408`
+which corresponds to `Job started and finished times.`
+
+As used in the previous section, the `upgrade` command without any
+additional argument will attempt to upgrade the database to the latest
+revision (`head`). But it is also possible to pass a revision hash in
+the event that you want to upgrade to a certain revision.
+
+Let us upgrade to the next revision (`Workflow complexity.`) by passing
+its revision hash (`f84e17bd6b18`) to test the database schema changes
+introduced there:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -c rest-api -- reana-db alembic upgrade f84e17bd6b18
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+```
+
+In some exceptional cases you might need to `downgrade` to a previous
+revision. For example, due to a failure during the release upgrade
+process, or because you were testing an alpha version and you want to
+revert to the previous state. Imagine that something went wrong in the
```

Comment:
Minor: this sentence which starts with "Imagine.." is kind of duplicate with a previous one IMO. Since in the previous sentence you already explained that one may want to downgrade in case of failure or testing alpha